### ISSUE TITLE
Clickable option to open folder from grid

### ISSUE THREAD

**Issue by ciar4n** [joomla-projects/media-manager-improvement#217]
Currently folders are opened with a double click. I'd suggest a single click option. Possibly an open folder icon similar to the delete/download/edit icons on image items

**Comment by laoneo** [comment]
I guess a single click would be enough, never saw an open icon in an app. But perhaps I'm wrong.

**Comment by kasvith** [comment]
But most people are comfortable with double click.
We can add this to preferences so user can try any method, for example even in google drive double click is used to open a folder

**Comment by laoneo** [comment]
I guess it is not bad to orienting ourselves a bit on Google.

**Comment by ciar4n** [comment]
Single click currently brings up the item info which is why I suggested the icon. Whatever you choose I would suggest having the same for both files and folders to bring up the item info.

**Comment by laoneo** [comment]
Agree

**Comment by ciar4n** [comment]
Double click could still open the folder. I just think a second one click option is needed as not everyone would think of double clicking within a browser window.

**Comment by laoneo** [comment]
In drive the single click opens the info panel, I think this would not be bad to do it the same in mm. Guess everybody is used to double click on a folder to go into that folder.

**Comment by ciar4n** [comment]
> Guess everybody is used to double click on a folder to go into that folder.

True for outside of the browser but I personally wouldn't agree for inside the browser. Considering you need JS for any double click events indicates it is not 'normal' behavior for inside the browser IMO.

What you suggest is how it is currently so this can be closed.

**Comment by laoneo** [comment]
Yes sorry was confused. True you suggested something else. I would like to keep it open, perhaps we get more opinions.

**Comment by dneukirchen** [comment]
I think users get used to dblclick in media manager pretty quick. We need the single click for selection, so i prefer

* folder
    * single click: select
    * dbl click: open
* file 
   * single click: select
   * dbl click: preview

This is UX. We should add this issue to the document dawn created some weeks ago.

**Comment by kasvith** [comment]
@dneukirchen that was the best solution for me so far

**Comment by brianteeman** [comment]
Personally double click is completely non-intuitive for anything browser based

**Comment by brianteeman** [comment]
I want to raise this again. Double click is not a pattern used on the web for anything anywhere so it is counter intuitive to use it here.

I know I raised it before but I cant find it and it makes sense to add it here as well.
The way you select multiple items is also not standard. 
I would expect to multiple items to be able to click on the first image and then shift click on the last image to select them all and to be able to ctrl click to deselect one of those images etc

And of course you must be able to do this in an accessible way as well using the expected methods

**Comment by schnuti** [comment]
@brianteeman To split click and double click you need two clickable areas. I find it ok as is for images. If you hover the image and 1. click on the dropdown to open it. 2. click on the first icon opens the modal.
My proposal would be to add this icon to the folder dropdown with the action open/view this folder.
In the list view the lines could be split up. 1. Click on icon to select 2. click on the rest opens the image modal/the folder. I'll add some proposal issues e.g. add the dropdown to the listview.

> The way you select multiple items is also not standard.

The Media Manager should follow the Joomla core handling.

**Comment by schnuti** [comment]
#555

**Comment by schnuti** [comment]
Wrong button
